fail2ban/server/action.py

```diff
@@ -532,7 +532,7 @@ def _processCmd(self, cmd, aInfo = None):
 		return self.executeCmd(realCmd, self.timeout)
 
 	@staticmethod
-	def executeCmd(realCmd, timeout=60):
+	def executeCmd(realCmd, timeout=60, testCmd=False):
```

Comment:
I used this name to expose that this command is a test on a condition; may be a better name?